content/docs/refs-and-the-dom.md

```diff
@@ -31,30 +31,61 @@ Your first inclination may be to use refs to "make things happen" in your app. I
 
 ### Adding a Ref to a DOM Element
 
-React supports a special attribute that you can attach to any component. The `ref` attribute takes a callback function, and the callback will be executed immediately after the component is mounted or unmounted.
+>**Note:**
+>
+>The examples below have updated to use the new `React.createRef()` API, introduced in React 16.3.
 
-When the `ref` attribute is used on an HTML element, the `ref` callback receives the underlying DOM element as its argument. For example, this code uses the `ref` callback to store a reference to a DOM node:
 
-```javascript{8,9,19}
+Refs can be created using `React.createRef()` and are commonly assigned to a component class instance in the constructor or via a property initializer.
```

Comment:
It's a bit hard to grok this initial example without context. How about leading with the second one that includes the render method?

Even better may be the canonical "autofocus an input" example that shows how to use a ref end-to-end.

Reply:
Yeah, we could keep the sentences separate (I like them) but maybe just have one code example (that highlights both lines)

Reply:
> commonly assigned to a **component class instance** in the constructor or via a property initializer.

Shouldn't it be instance _property_?

> Refs can be created using `React.createRef()` and are commonly assigned to an instance property when a component is constructed. _(Avoids going into class property vs constructor details)._

Reply:
I dig that.